docs/content/use_cases/items/question_the_docs.md

```diff
@@ -8,44 +16,68 @@ os.environ['OPENAI_API_KEY'] = '<YOUR-OPENAI-API-KEY>'
 
 
 ```python
+import os
 from superduperdb import superduper
 from superduperdb.db.mongodb.query import Collection
-import pymongo
 
-db = pymongo.MongoClient().documents
-db = superduper(db)
+# Uncomment one of the following lines to use a bespoke MongoDB deployment
+# For testing the default connection is to mongomock
+
+mongodb_uri = os.getenv("MONGODB_URI","mongomock://test")
+# mongodb_uri = "mongodb://localhost:27017"
+# mongodb_uri = "mongodb://superduper:superduper@mongodb:27017/documents"
+# mongodb_uri = "mongodb://<user>:<pass>@<mongo_cluster>/<database>"
+# mongodb_uri = "mongodb+srv://<username>:<password>@<atlas_cluster>/<database>"
+
+# Super-Duper your Database!
+from superduperdb import superduper
+db = superduper(mongodb_uri)
 
 collection = Collection('questiondocs')
 ```
 
+In this example we use the internal textual data from the `superduperdb` project's API documentation, with the "meta"-goal of 
+creating a chat-bot to tell us about the project which we are using!
+
+
+```python
+import glob
+glob.glob('../superduperdb/docs/content/docs/*/*.md')
+```
+
 
 ```python
 import glob
 
 STRIDE = 5       # stride in numbers of lines
 WINDOW = 10       # length of window in numbers of lines
 
-content = sum([open(file).readlines() for file in glob.glob('../*/*.md') + glob.glob('../*.md')], [])
+content = sum([open(file).readlines() 
+               for file in glob.glob('../superduperdb/docs/content/docs/*/*.md') 
+               + glob.glob('../superduperdb/docs/content/docs/*.md')], [])
```

Comment:
I wonder if it's to reference files or use GitHub links.

If we reference files, then the notebooks are not standalone and require the whole project to be locally available.